packages/dds/tree/src/shared-tree/treeCheckout.ts

```diff
@@ -913,3 +909,22 @@ export class TreeCheckout implements ITreeCheckoutFork {
 		);
 	}
 }
+
+// Keeps track of all new forks created until the returned function is invoked, which will dispose all of those forks
```

Comment:
nit: one of the benefits of factoring code out into helper functions is that you can put doc-comments on them. So let's take advantage of that and make this a full-on doc comment.

Reply:
added doc comment to helper function